src/BenchmarkDotNet/Reports/Measurement.cs

```diff
@@ -61,19 +62,44 @@ public Measurement(int launchIndex, IterationMode iterationMode, IterationStage
             IterationIndex = iterationIndex;
         }
 
-        public string ToOutputLine()
```

Comment:
nit: imho the old name was more clear about the intentions

Reply:
Existing of an additional `ToOutputLine` method led to some bugs because we had two different string representations of `Measurement`. Currently, we have a single unified CultureInfo-independent representation in all the places.